scripts/perftests/testCases/OrchardCore-5dbd92c.ps1

```diff
@@ -20,7 +20,8 @@ if ($globalJsonPath) {Remove-Item "$sourcePath\$globalJsonPath"}
 
 #########################################################
 # https://github.com/G-Research/NuPerfMonitor/issues/8
-Set-Content -Path "$sourcePath\MSBuild.rsp" -Value "-p:EnableSdkContainerSupport=false"
```

Comment:
is `-p:EnableSdkContainerSupport=false` not needed anymore? If it's not needed, I guess the comment above can be removed.